packages/cli/test/test-kit/cli-test-kit.ts

```diff
@@ -1,10 +1,11 @@
 import { readdirSync, readFileSync, statSync, writeFileSync, existsSync, mkdirSync } from 'fs';
-import { spawn, ChildProcessWithoutNullStreams, spawnSync } from 'child_process';
```

Comment:
changes in `cli-test-kit` are not directly related to the this PR. maybe add some comment in the PR description to indicate this change